plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/FacetUtil.java

```diff
@@ -132,7 +132,7 @@ public FacetUtil addWebFacetToBatch(IProjectFacetVersion webFacet) {
     IDataModel webModel = DataModelFactory.createDataModel(new WebFacetInstallDataModelProvider());
     webModel.setBooleanProperty(IJ2EEModuleFacetInstallDataModelProperties.ADD_TO_EAR, false);
     webModel.setBooleanProperty(IJ2EEFacetInstallDataModelProperties.GENERATE_DD, true);
-    webModel.setBooleanProperty(IWebFacetInstallDataModelProperties.INSTALL_WEB_LIBRARY, false);
+    webModel.setBooleanProperty(IWebFacetInstallDataModelProperties.INSTALL_WEB_LIBRARY, true);
```

Comment:
Is it worth setting `INSTALL_WEB_LIBRARY` too? (I think it defaults to true.)
Or if `INSTALL_WEB_LIBRARY` is true by default, should we remove it here?

(I'm thinking being more explicit is useful?)